pkg/utils/installer/install.go

```diff
@@ -431,6 +431,10 @@ func CreateDaemonSet(ctx context.Context, identity string, directCSIContainerIma
 				}(),
 				SecurityContext: &corev1.SecurityContext{
 					Privileged: &privileged,
+					SeccompProfile: &corev1.SeccompProfile{
```

Comment:
Should we enable Seccomprofile by default?

Reply:
No. Made it optional now.